tests/wpt/meta/streams/piping/throwing-options.any.js.ini

```diff
@@ -20,22 +20,8 @@
   expected: ERROR
 
 [throwing-options.any.html]
-  expected: TIMEOUT
-  [pipeTo should stop after getting signal throws]
-    expected: TIMEOUT
-
-  [pipeThrough should stop after getting signal throws]
-    expected: FAIL
-
 
 [throwing-options.any.worker.html]
-  expected: TIMEOUT
-  [pipeTo should stop after getting signal throws]
-    expected: TIMEOUT
-
-  [pipeThrough should stop after getting signal throws]
-    expected: FAIL
-
```

Comment:
@Taym95 I guess we have our answer re https://github.com/servo/servo/pull/37245#pullrequestreview-2899404498